testing/trino-tests/src/test/java/io/trino/execution/TestQueues.java

```diff
@@ -300,6 +300,20 @@ public void testSelectorResourceEstimateBasedSelection()
         }
     }
 
+    @Test
+    @Timeout(240)
```

Comment:
do we need timeout that long?

Reply:
All tests in this class have this timeout

Reply:
Is it needed? :)